frontend/src/scenes/plugins/edit/PluginSourceTabs.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { pluginSourceLogic } from 'scenes/plugins/edit/pluginSourceLogic'
+import { LemonButton } from 'lib/components/LemonButton'
+import { LemonRow } from 'lib/components/LemonRow'
+
+export function PluginSourceTabs(): JSX.Element {
+    const { setCurrentFile } = useActions(pluginSourceLogic)
+    const { currentFile, fileNames, pluginSourceValidationErrors } = useValues(pluginSourceLogic)
+
+    return (
+        <LemonRow style={{ padding: 0 }}>
```

Comment:
This doesn't need to be a `LemonRow`, especially since the child buttons are `LemonRow`-based already. We just need flexbox.

Reply:
This was even in storybook, so 🙈. Thanks!